dev/composer.ts

```diff
@@ -33,6 +33,66 @@ interface ComposerListOutput {
   commands: ComposerCommand[];
 }
 
+const PACKAGE_REGEXP = new RegExp("^.*/.*$");
+
+const searchGenerator: Fig.Generator = {
+  script: function (context) {
+    if (context[context.length - 1] === "") return "";
+    const searchTerm = context[context.length - 1];
+    return `curl -s -H "Accept: application/json" "https://packagist.org/search.json?q=${searchTerm}&per_page=20"`;
+  },
+  postProcess: function (out) {
+    try {
+      var temp = JSON.parse(out).results.map(
+        (item) =>
+          ({
+            name: item.name,
+            description: item.description,
+            icon: "📦",
+          } as Fig.Suggestion)
+      ) as Fig.Suggestion[];
+    } catch (e) {
+      return [];
+    }
+
+    return temp;
```

Comment:
We can remove the `var` and simplify a bit:

```suggestion
    try {
      return JSON.parse(out).results.map(
        (item) =>
          ({
            name: item.name,
            description: item.description,
            icon: "📦",
          } as Fig.Suggestion)
      ) as Fig.Suggestion[];
    } catch (e) {
      return [];
    }
```